components/about.tsx

```diff
@@ -1,81 +1,101 @@
-              fill="url(#illustration-02)"
-            />
-          </svg>
-        </div>
+import React from "react";
+import { Card, CardContent, CardDescription } from "@/components/ui/card";
 
-        {/* Hero content */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
-          {/* Section header */}
-          <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Run by nang and FryingPan
-            </h1>
-            <p
-              className="mb-8 text-xl text-gray-600"
-              data-aos="fade-up"
-              data-aos-delay="200"
-            >
-              Building in Public
+const AboutComponent: React.FC = () => {
+  return (
+    <section
+      className="to-white bg-gradient-to-b from-gray-50 px-4 py-12 pt-20"
+      data-aos="fade-up"
+      data-aos-delay="200"
+    >
+      <div className="mx-auto max-w-6xl">
+        <div className="flex flex-col lg:flex-row">
+          {/* Left column */}
+          <div className="lg:w-1/2 lg:pr-8">
```

Comment:
```suggestion
          <div className="lg:w-1/2 lg:pr-8 lg:pt-4">
```

This adds padding to the top of the left column so it is aligned with the video card

Before:
![image](https://github.com/trypear/pear-landing-page/assets/87584757/f3da0b9a-a09b-410a-a3ff-d8d0312faace)

After:
![image](https://github.com/trypear/pear-landing-page/assets/87584757/c812c903-b044-41db-a2d6-8ca47d9c75c6)